test/e2e/nodeportlocal_test.go

```diff
@@ -245,10 +332,23 @@ func deleteNPLRuleFromIPTables(t *testing.T, data *TestData, r *require.Assertio
 	r.NoError(err, "Error when deleting iptables rule")
 }
 
+func deleteNPLRuleFromNetNat(t *testing.T, data *TestData, r *require.Assertions, antreaPod string, rule nplRuleData) {
+	t.Logf("Deleting Netnat rule for %v", rule)
+	const timeout = 30 * time.Second
+	err := wait.Poll(time.Second, timeout, func() (bool, error) {
+		_, _, _, err := data.RunCommandOnNode(rule.nodeIP, "Remove-NetNatStaticMapping -NatName antrea-nat -StaticMappingID 1 -Confirm:$false")
```

Comment:
Why use retry here? Why is it possible to fail in the first delete, but succeed in the second try?

Reply:
To avoid ssh failure for RunCommandOnNode function.